dspace-api/src/main/resources/Messages.properties

```diff
@@ -2391,6 +2391,10 @@ jsp.mydspace.cris.rp-status-change.keep-active = Keep the profile public
 jsp.mydspace.cris.rp-status-change.keep-inactive = Keep the profile private
 jsp.mydspace.cris.rp-status-change.keep-undefined = Decide later
 
+jsp.layout.cris.crisrp.sections.profile.title = Profile
```

Comment:
you can define the label of the "menu sections" in the messages.properties, "profile" is the name assigned in the cris-vertical-components.cfg file